### PR TITLE
feat: support overriding czml appearance on reearth/core

### PR DESCRIPTION
**Pull request by keiya01**
# Overview

I implemented overriding  czml appearance support on reearth/core.

### CODE REVIEW

File: src/core/engines/Cesium/Feature/Resource/index.tsx [hunk 1]
```diff
@@ -18,6 +27,12 @@ const comps = {
   geojson: GeoJsonDataSource,
 };
 
+const delegatingAppearance: Record<keyof typeof comps, (keyof AppearanceTypes)[]> = {
+  kml: [],
+  geojson: [],
+  czml: ["marker", "polyline", "polygon"],
```
Comment (rot1024) [inline src/core/engines/Cesium/Feature/Resource/index.tsx:33]:
```suggestion
  kml: ["marker", "polyline", "polygon"],
  geojson: ["marker", "polyline", "polygon"],
  czml: ["marker", "polyline", "polygon"],
```

**Review by rot1024 — APPROVED** [review]

File: src/core/engines/Cesium/Feature/Resource/utils.ts [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+          heightReference: {
+            name: "heightReference",
+            type: "heightReference",
+          },
+        });
+      }
+
+      if (appearance === "polyline") {
+        const entityPosition = entity.position?.getValue(JulianDate.now());
+        const positions = entity.polyline?.positions?.getValue(JulianDate.now()) as Cartesian3[];
```
Comment (rot1024) [inline src/core/engines/Cesium/Feature/Resource/utils.ts:155]:
```suggestion
        const positions = entity.polyline?.positions?.getValue() as Cartesian3[];
```

pass undefined or null for getValue. Actually It should be viewer's clock time, but for now we cannot update features by changing time.

**Review by rot1024 — APPROVED** [review]

File: src/core/engines/Cesium/Feature/Resource/index.tsx (hunk 1 above)
Comment (keiya01) [reply src/core/engines/Cesium/Feature/Resource/index.tsx]: I fixed.
[feat: support overriding style on kml](https://github.com/reearth/reearth-web/pull/421/commits/b78f20a8ff684094a05681f2397f8d63fa2c59ae)

File: src/core/engines/Cesium/Feature/Resource/utils.ts (hunk 2 above)
Comment (keiya01) [reply src/core/engines/Cesium/Feature/Resource/utils.ts:157]: I fixed.
https://github.com/reearth/reearth-web/pull/421/commits/482074685ea757a1a12461580cb84f6817960afa

File: src/core/engines/Cesium/Feature/Resource/utils.ts [hunk 3]
```diff
@@ -100,6 +101,7 @@ export const attachStyle = (
   appearances: (keyof AppearanceTypes)[] | undefined,
   layer: ComputedLayer | undefined,
   evalFeature: EvalFeature,
+  viewer: Viewer,
```
Comment (rot1024) [inline src/core/engines/Cesium/Feature/Resource/utils.ts:104]: IMO

```suggestion
  currentTime: JullianDate,
```

**Review by rot1024 — APPROVED** [review]

**Pull request merged**